### PR TITLE
2022-01-08 Matt Butcher

### PR DESCRIPTION
**Pull request by allenwyma**

### CODE REVIEW

File: _episodes/interviews/2022-12-23-matt-butcher.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Fermyon with Matt Butcher"
+date: 2022-12-23T16:51:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2022-12-23-matt-butcher.mp3
+duration: "54:19"
+length: "52142916"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Matt Butcher](https://twitter.com/technosophos), CEO of [Fermyon](https://www.fermyon.com/), who is working on [Fermyon Cloud](https://www.fermyon.com/cloud), a platform to effortlessly deploy and host your WebAssembly applications.
```
Comment (jonhoo) [inline _episodes/interviews/2022-12-23-matt-butcher.md:9]:
```suggestion
Allen Wyma talks with [Matt Butcher](https://twitter.com/technosophos), CEO of [Fermyon](https://www.fermyon.com/), who is working on [Fermyon Cloud](https://www.fermyon.com/cloud), a platform to deploy and host WebAssembly applications.
```
Let's not do a sales pitch in the notes :)

Also, what's the connection to Rust here?

File: _episodes/interviews/2022-12-23-matt-butcher.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+- [@0:00] - Introduction
+- [@2:13] - A deeper dive into the Cloud and its impact
+- [@9:39] - Matt's work experience on HP and HP Cloud.
+- [@13:40] - Kubernetes
+- [@20:07] - WebAssembly
+- [@30:27] - What happened to Krustlet
```
Comment (jonhoo) [inline _episodes/interviews/2022-12-23-matt-butcher.md:26]: What is Krustlet?

File: _episodes/interviews/2022-12-23-matt-butcher.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Fermyon with Matt Butcher"
+date: 2022-12-23T16:51:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2022-12-23-matt-butcher.mp3
```
Comment (jonhoo) [inline _episodes/interviews/2022-12-23-matt-butcher.md:4]: We should update the dates for these before release.

File: _episodes/interviews/2022-12-23-matt-butcher.md (hunk 1 above)
Comment (allenwyma) [reply _episodes/interviews/2022-12-23-matt-butcher.md]: @jonhoo let me know if this suffices:

Fermyon is nearly all written in Rust and WebAssembly has first class Rust support. Not to mention, Matt has extensive use of Rust with his involvement in Krustlet and various other projects written in Rust as we discuss within the episode.

Comment (jonhoo) [reply _episodes/interviews/2022-12-23-matt-butcher.md]: @allenwyma Yes, that's great thanks!

**Review by jonhoo — APPROVED** [review]

**Pull request merged**